apps/web/src/server/views/app/dashboard.njk

```diff
@@ -4,41 +4,29 @@
 {% set pageTitle = 'Back office services' %}
 
 {% block main %}
```

Comment:
this whole page can go now - users won't be taken here anymore. `/applications-service/` is the new common landing page